### DISCUSSION TITLE
User is typing..

### DISCUSSION THREAD

**Discussion by malikdoksoz** [flyerhq/flutter_chat_ui#191] [Ideas & Requests]
How can I show in chat while the **_user is typing_** a message? 

![image_thumb%255B2%255D](https://user-images.githubusercontent.com/26676061/148816714-719e589c-e90e-4c80-9646-373358640082.png)

**Comment by demchenkoalex** [comment]
Oops forgot about that. To be added.

**Reply by malikdoksoz** [reply]
Did you forget that it didn't come in the new version? 😊

**Reply by demchenkoalex** [reply]
What? :D where did I say it was included in the new version?

**Reply by malikdoksoz** [reply]
Oh no you didnt say, I just thought it would be included 😊 

If you dont have time I can open a pr for this. Good work 😊

**Reply by demchenkoalex** [reply]
Yeah I will not be working on that at the moment. This is what our designer prepared some time ago, don't know about moving dots, I think if they are static just with blinking opacity that is okay. When this appears scroll view with messages should be pushed up with animation. And code should be without any dependencies. If you want you can take a shot.

<img width="399" alt="Screenshot 2022-01-21 at 19 40 22" src="https://user-images.githubusercontent.com/14123304/150582308-f2366b62-148c-42bd-b2b8-f419aeff23b8.png">

**Reply by mattmattkim** [reply]
Great library! Curious if this feature has been implemented. I couldn't find it in the doc.

**Reply by demchenkoalex** [reply]
It is on a `main` branch, but not released

**Reply by Phuong-Tran-it** [reply]
> It is on a `main` branch, but not released

I also have the same problem

**Comment by demchenkoalex** [comment]
make sure to add customisation https://github.com/flyerhq/flutter_chat_ui/discussions/683

**Comment by hajajmaor** [comment]
Any update?

**Reply by demchenkoalex** [reply]
What update? It is there
